src/sim65/memory.c

```diff
@@ -64,6 +64,15 @@ void MemWriteByte (unsigned Addr, unsigned char Val)
 
 
 
+void MemWriteWord (unsigned Addr, unsigned Val)
```

Comment:
I had deliberately not put this here as a 65xx never writes a word meaning that this is conceptually always limited to paravirt stuff. But I'm as well okay with it going here.